src/MUIDataTable.js

```diff
@@ -200,6 +200,7 @@ class MUIDataTable extends React.Component {
       onDownload: PropTypes.func,
       setTableProps: PropTypes.func,
       setRowProps: PropTypes.func,
+      selectToolbarPlacement: PropTypes.string,
```

Comment:
Let's use the `oneOf` for the prop types definition, as we do for other "enums."